README.md

```diff
@@ -452,6 +452,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Kabir Jain](https://github.com/kkkkkabir)
   - [Kaiwalya Koparkar](https://github.com/kaiwalyakoparkar)
   - [Kanishk Pachauri](https://github.com/Mr-Sunglasses)
+  - [karan kulshrestha](https://github.com/karankulshrestha)
```

Comment:
Might want to capitalize your name 😅 What do you think?
```suggestion
  - [Karan Kulshrestha](https://github.com/karankulshrestha)
```

Reply:
Maybe you are right but this looks also good 🙂